earth2studio/data/xr.py

```diff
@@ -74,14 +70,9 @@ class DataSetFile:
         Data array name in xarray dataset
     """
 
-    def __init__(self, file_path: str, array_name: str):
+    def __init__(self, file_path: str, array_name: str, **xr_args: Any):
```

Comment:
could this be `backend_kwargs: dict{str, Any]` so it matches the other IO classes?

Also doc string needs a quick update.

Reply:
Yes! Good point.

Reply:
This is tricky because technically these keyword arguments go directly to xarray's `open_dataset` and among those keyword arguments is an argument explicitly labeled as `backend_kwargs`. I am worried about users misinterpreting the argument names here.